### ISSUE TITLE
App subtitles are very small and with a black background-box no matter the options 

### ISSUE THREAD

**Issue by hectorjroman** [jellyfin/jellyfin-androidtv#2699]
### Describe the bug

All the subtitles are shown with very small letters and with a black background box no matter the settings in the App.

### Media info of the file

```
General
ID única                                 : 134076075871645204552770151759265390615 (0x64DE1CE94A15B1B9F99D2CE8E647E017)
Nombre completo                          : [redacted for privacy reasons].mkv
Formato                                  : Matroska
Formato de la versión                    : Version 4
Tamaño de archivo                        : 3,28 GiB
Duración                                 : 1 h 41 min
Tasa de bits general                     : 4 613 kb/s
Velocidad de fotogramas                  : 23,976 FPS
Fecha de codificación                    : 2023-01-26 18:08:30 UTC
Aplicación de codifición                 : mkvmerge v73.0.0 ('25 or 6 to 4') 64-bit
Librería de codificación                 : libebml v1.4.4 + libmatroska v1.7.1

Vídeo
ID                                       : 1
Formato                                  : AVC
Formato/Info                             : Advanced Video Codec
Formato del perfil                       : High@L4
Ajustes del formato                      : CABAC / 8 Ref Frames
Ajustes del formato, CABAC               : Sí
Ajustes del formato, RefFrames           : 8 fotogramas
ID códec                                 : V_MPEG4/ISO/AVC
Duración                                 : 1 h 41 min
Tasa de bits                             : 4 000 kb/s
Ancho                                    : 1 280 píxeles
Alto                                     : 720 píxeles
Relación de aspecto                      : 16:9
Modo velocidad fotogramas                : Constante
Velocidad de fotogramas                  : 23,976 (24000/1001) FPS
Espacio de color                         : YUV
Submuestreo croma                        : 4:2:0
Profundidad bits                         : 8 bits
Tipo barrido                             : Progresivo
Bits/(píxel*fotograma)                   : 0.181
Tamaño de pista                          : 2,74 GiB (83%)
Librería de codificación                 : x264 core 164 r3094 bfc87b7
Opciones de codificación                 : cabac=1 / ref=8 / deblock=1:-1:-1 / analyse=0x3:0x133 / me=umh / subme=9 / psy=1 / psy_rd=1.00:0.15 / mixed_ref=1 / me_range=16 / chroma_me=1 / trellis=2 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-3 / threads=8 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / stitchable=1 / constrained_intra=0 / bframes=3 / b_pyramid=2 / b_adapt=2 / b_bias=0 / direct=3 / weightb=1 / open_gop=0 / weightp=2 / keyint=48 / keyint_min=4 / scenecut=0 / intra_refresh=0 / rc_lookahead=48 / rc=abr / mbtree=1 / bitrate=4000 / ratetol=1.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / vbv_maxrate=4400 / vbv_bufsize=8000 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=1:1.00
Idioma                                   : Inglés
Default                                  : Sí
Forced                                   : No
Rango de color                           : Limited
Colores primarios                        : BT.709
Características transferencia            : BT.709
Coeficientes matriz                      : BT.709

Audio
ID                                       : 2
Formato                                  : E-AC-3 JOC
Formato/Info                             : Enhanced AC-3 with Joint Object Coding
Nombre comercial                         : Dolby Digital Plus with Dolby Atmos
ID códec                                 : A_EAC3
Duración                                 : 1 h 41 min
Tipo de tasa de bits                     : Constante
Tasa de bits                             : 768 kb/s
Canal(es)                                : 6 canales
Channel layout                           : L R C LFE Ls Rs
Velocidad de muestreo                    : 48,0 kHz
Velocidad de fotogramas                  : 31,250 FPS (1536 SPF)
Modo de compresión                       : Con pérdida
Tamaño de pista                          : 560 MiB (17%)
Idioma                                   : Inglés
Service kind                             : Complete Main
Default                                  : Sí
Forced                                   : No
Complexity index                         : 16
Number of dynamic objects                : 15
Bed channel count                        : 1 canal
Bed channel configuration                : LFE

Texto
ID                                       : 3
Formato                                  : UTF-8
ID códec                                 : S_TEXT/UTF8
ID códec/Info                            : UTF-8 Plain Text
Duración                                 : 1 h 41 min
Tasa de bits                             : 69 b/s
Velocidad de fotogramas                  : 0,271 FPS
Count of elements                        : 1648
Tamaño de pista                          : 51,4 KiB (0%)
Idioma                                   : Inglés
Default                                  : Sí
Forced                                   : No
```

### Application version

0.15.7

### Device information

Google Chromecast HD with Google TV

### Android version

Android TV 12

### Jellyfin server version

10.8.10

**Comment by nielsvanvelzen** [comment]
No details like media info makes it hard to help

**Comment by jimmyjones865** [comment]
I have a similar issue, my subtitles are very large, but also have a black background and the appearance can't be changed via settings. This is new since this version. A few days ago, I watched some episodes with embedded subrip subtitles that had correct formatting according to my settings. With this version, the same subtitles in the same episode files are now large and with a black background.

**Comment by hectorjroman** [comment]
> No details like media info makes it hard to help

It happens with all files.

**Comment by hectorjroman** [comment]
> It happens with all media types with embedded and external SRT, and possibly other text type subtitles. Previous version did not have this problem.

Exactly. On Saturday 22 it worked fine and after the update on Sunday 23, the same movie I watched fine the day before had the problem.

**Comment by hectorjroman** [comment]
> Same here on Shield TV, only with subtitles too large on black background. Unusable right now.

Unusable, indeed. I had to fall back to Plex in the meantime until the issue is solved.

**Comment by nielsvanvelzen** [comment]
Once again, without proper details we cannot reproduce the issue nor fix it. There is an issue template (that was not filled in!) with a reason.

**Comment by nielsvanvelzen** [comment]
Thanks, should be fixed with #2710.

**Comment by douglasg14b** [comment]
Am I missing what the fix was here? Been noticing this last year and this year still 🤔

Android TV App
